packages/oc-s3-riak-storage-adapter/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "oc-s3-riak-storage-adapter",
+  "version": "1.0.0",
```

Comment:
Could you put the version to 0.0.1 so that is not yet a 1.0 release when we publish it?